ansible/roles/prosody/templates/prosody.cfg.lua.j2

```diff
@@ -55,6 +55,7 @@ modules_enabled = {
       "adhoc";
       "websocket";
       "bosh";
+      "muc_get_media_type";
```

Comment:
You are adding this as global module, this means it will be attached to every virtual host that is defined, do we need it to all virtual hosts that are configured?

Reply:
good point - I will check it - thank you @damencho

Reply:
no needed - I've removed it